### ISSUE TITLE
关于文本生成

### ISSUE THREAD

**Issue by yuzhang112** [dbiir/UER-py#219]
你好，我想用自己的语料，对预训练GPT2模型进行微调，然后进行文本生成任务，请问可以用增量预训练的命令进行微调吗？

这是我现在使用的预训练命令：
python3 pretrain.py --dataset_path dataset.pt --pretrained_model_path models/cluecorpussmall_gpt2_seq1024_model.bin-250000 --vocab_path models/google_zh_vocab.txt --config_path models/gpt2/distil_config.json --output_model_path models/my_gpt2_model.bin --world_size 1 --gpu_ranks 0 --total_steps 10000 --save_checkpoint_steps 5000 --report_steps 1000  --learning_rate 5e-5 --batch_size 32  --embedding word_pos --remove_embedding_layernorm --encoder transformer --mask causal --layernorm_positioning pre  --target lm --tie_weights

**Comment by yuzhang112** [comment]
好的，感谢回复
